tests/test_astroid.py

```diff
@@ -1,15 +1,38 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals, print_function
+
+import re
+
 import astroid
 from . import tools, test_mark_tokens
 
 
 class TestAstroid(test_mark_tokens.TestMarkTokens):
 
   is_astroid_test = True
+  module = astroid
 
   @classmethod
   def create_mark_checker(cls, source):
     builder = astroid.builder.AstroidBuilder()
     tree = builder.string_build(source)
     return tools.MarkChecker(source, tree=tree)
+
+  def assert_nodes_equal(self, node1, node2):
+    self.assertEqual(
+      repr_tree(node1),
+      repr_tree(node2),
+    )
+
+
+def repr_tree(node):
+  result = node.repr_tree()
+  result = result.replace('AssignName(name=', 'Name(name=')
+  result = result.replace('DelName(name=', 'Name(name=')
```

Comment:
I'd prefer consistency here, i.e. make this similar to the next line (since they are solving essentially identical problem). And definitely explain in a comment. I looked into it, and if my understanding is correct, something along these lines perhaps:

Treat AssignName, DelName, and Name as equal, since the indicate a difference in context rather than in the text of the node. E.g. "a" is an Name in expression "a + 1" and is an AssignName in expression "a = 1".